docs/CONTRIBUTING.md

```diff
@@ -61,25 +61,17 @@ Here are a few things you can do that will increase the likelihood of your pull
 Supply a file glob to the test command:
 
 ```command
-TEST="test/components/YOUR_COMPONENT_test.rb" bundle exec rake
+TEST="test/components/YOUR_COMPONENT_test.rb" bundle exec appraisal rake
 ```
 
-### Run tests for a specific version of Rails
-
-Update the bundle:
-
-```command
-RAILS_VERSION=5.2.5 bundle update
-```
+### Running tests for a specific version of Rails
 
-Then run the test command:
+Specify one of the supported versions, listed in [Appraisal](https://github.com/github/view_component/blob/main/Appraisals):
 
 ```command
-RAILS_VERSION=5.2.5 bundle exec rake
+bundle exec appraisal rails-5.2 rake
 ```
 
-When you're done, make sure you don't commit changes to `Gemfile.lock`. Instead, discard your changes to the file: `git checkout -- Gemfile.lock`
```

Comment:
I figure this was removed because changes to ViewComponent's gem dependencies in test should only change the Appraisal `Gemfile.lock`s, which are ignored. Is that correct?

Reply:
Yes, exactly. It's much cleaner this way.